packages/connect-core/src/utils/app.ts

```diff
@@ -8,6 +8,10 @@ export const apmAppId = (appName: string): string =>
   ethersUtils.namehash(`${appName}.aragonpm.eth`)
 
 function signatureFromAbi(signature: string, abi: Abi): string {
+  if (signature === 'fallback') {
+    return '()'
```

Comment:
What do you think of returning the whole signature here? I did a similar fix on the 1Hive fork and I remember we had some issues with radspec descriptions otherwise.